### ISSUE TITLE
Bitcoin core logs not working

### ISSUE THREAD

**Issue by schjonhaug** [vulpemventures/nigiri#141]
Hi, I can‘t seem to get any logs from bitcoin core. `electrs` and `chopsticks` are working. However, `node` doesn‘t produce any output.

```
❯ nigiri logs node

❯ nigiri logs electrs
electrs  | Config { log: StdErrLog { verbosity: Trace, quiet: false, show_level: true, timestamp: Off, modules: [], writer: "stderr", color_choice: Never }, network_type: Regtest, db_path: "./db/regtest", daemon_dir: "/config/regtest", blocks_dir: "/config/regtest/blocks", daemon_rpc_addr: 172.27.0.2:18443, cookie: Some("admin1:123"), electrum_rpc_addr: 0.0.0.0:50000, http_addr: 0.0.0.0:30000, http_socket_file: None, monitoring_addr: 127.0.0.1:24224, jsonrpc_import: true, light_mode: false, address_search: false, index_unspendables: false, cors: Some("*"), precache_scripts: None, utxos_limit: 500, electrum_txs_limit: 500, electrum_banner: "Welcome to electrs-esplora 0.4.1" }
electrs  | DEBUG - Server listening on 127.0.0.1:24224
electrs  | DEBUG - Running accept thread
electrs  | WARN - reconnecting to bitcoind: {"code":-28,"message":"Loading block index…"}
electrs  | INFO - NetworkInfo { version: 220000, subversion: "/Satoshi:22.0.0/", relayfee: 0.00001 }
electrs  | INFO - BlockchainInfo { chain: "regtest", blocks: 0, headers: 0, bestblockhash: "0f9188f13cb7b2c71f2a335e3a4fc328bf5beb436012afca590b1a11466e2206", pruned: false, verificationprogress: 1.0, initialblockdownload: Some(true) }
electrs  | WARN - waiting for bitcoind sync to finish: 0/0 blocks, verification progress: 100.000%
electrs  | DEBUG - opening DB at "./db/regtest/newindex/txstore"
electrs  | DEBUG - 0 blocks were added
electrs  | DEBUG - opening DB at "./db/regtest/newindex/history"
electrs  | DEBUG - 0 blocks were indexed
electrs  | DEBUG - opening DB at "./db/regtest/newindex/cache"
electrs  | DEBUG - downloading all block headers up to 24fa86051fa6efbd3c814394f96a4ed203e6fb57b194c4bb924e27eae6b8fe25
electrs  | TRACE - downloading 8 block headers
electrs  | INFO - hash=24fa86051fa6efbd3c814394f96a4ed203e6fb57b194c4bb924e27eae6b8fe25 height=7 @ 2022-03-25T19:24:02Z (8 left to index)
electrs  | DEBUG - adding transactions from 8 blocks using Bitcoind
electrs  | DEBUG - hash=24fa86051fa6efbd3c814394f96a4ed203e6fb57b194c4bb924e27eae6b8fe25 height=7 @ 2022-03-25T19:24:02Z (8 left to index)
electrs  | DEBUG - writing 56 rows to RocksDB { path: "./db/regtest/newindex/txstore" }, flush=Disable
electrs  | DEBUG - starting full compaction on RocksDB { path: "./db/regtest/newindex/txstore" }
electrs  | DEBUG - finished full compaction on RocksDB { path: "./db/regtest/newindex/txstore" }
electrs  | DEBUG - indexing history from 8 blocks using Bitcoind
electrs  | DEBUG - hash=24fa86051fa6efbd3c814394f96a4ed203e6fb57b194c4bb924e27eae6b8fe25 height=7 @ 2022-03-25T19:24:02Z (8 left to index)
electrs  | DEBUG - writing 16 rows to RocksDB { path: "./db/regtest/newindex/history" }, flush=Disable
electrs  | DEBUG - starting full compaction on RocksDB { path: "./db/regtest/newindex/history" }
electrs  | DEBUG - finished full compaction on RocksDB { path: "./db/regtest/newindex/history" }
electrs  | DEBUG - flushing to disk
electrs  | DEBUG - updating synced tip to 24fa86051fa6efbd3c814394f96a4ed203e6fb57b194c4bb924e27eae6b8fe25
electrs  | DEBUG - applying 8 new headers from height 0
electrs  | INFO - Electrum RPC server running on 0.0.0.0:50000
electrs  | INFO - REST server running on 0.0.0.0:30000
electrs  | TRACE - registering event source with poller: token=Token(0), interests=READABLE | WRITABLE
electrs  | DEBUG - downloading new block headers (8 already indexed) from 4ddb633f1690ec99d99e412095a8ae7482a1a91ce28b88ecee0e0015e25be5c0
electrs  | TRACE - downloaded 94 block headers
electrs  | INFO - hash=4ddb633f1690ec99d99e412095a8ae7482a1a91ce28b88ecee0e0015e25be5c0 height=101 @ 2022-03-25T19:24:18Z (94 left to index)
electrs  | DEBUG - adding transactions from 94 blocks using Bitcoind
electrs  | DEBUG - hash=4ddb633f1690ec99d99e412095a8ae7482a1a91ce28b88ecee0e0015e25be5c0 height=101 @ 2022-03-25T19:24:18Z (94 left to index)
electrs  | DEBUG - writing 658 rows to RocksDB { path: "./db/regtest/newindex/txstore" }, flush=Enable
electrs  | DEBUG - indexing history from 94 blocks using Bitcoind
electrs  | DEBUG - hash=4ddb633f1690ec99d99e412095a8ae7482a1a91ce28b88ecee0e0015e25be5c0 height=101 @ 2022-03-25T19:24:18Z (94 left to index)
electrs  | DEBUG - writing 188 rows to RocksDB { path: "./db/regtest/newindex/history" }, flush=Enable
electrs  | DEBUG - updating synced tip to 4ddb633f1690ec99d99e412095a8ae7482a1a91ce28b88ecee0e0015e25be5c0
electrs  | DEBUG - applying 94 new headers from height 8
electrs  | DEBUG - downloading new block headers (102 already indexed) from 60ca66619d731320e24d52aea93f5fb702fd627be024b25bc4915aebefce9835
electrs  | TRACE - downloaded 1 block headers
electrs  | INFO - hash=60ca66619d731320e24d52aea93f5fb702fd627be024b25bc4915aebefce9835 height=102 @ 2022-03-25T19:27:10Z (1 left to index)
electrs  | DEBUG - adding transactions from 1 blocks using Bitcoind
electrs  | DEBUG - hash=60ca66619d731320e24d52aea93f5fb702fd627be024b25bc4915aebefce9835 height=102 @ 2022-03-25T19:27:10Z (1 left to index)
electrs  | DEBUG - writing 11 rows to RocksDB { path: "./db/regtest/newindex/txstore" }, flush=Enable
electrs  | DEBUG - indexing history from 1 blocks using Bitcoind
electrs  | DEBUG - hash=60ca66619d731320e24d52aea93f5fb702fd627be024b25bc4915aebefce9835 height=102 @ 2022-03-25T19:27:10Z (1 left to index)
electrs  | DEBUG - writing 6 rows to RocksDB { path: "./db/regtest/newindex/history" }, flush=Enable
electrs  | DEBUG - updating synced tip to 60ca66619d731320e24d52aea93f5fb702fd627be024b25bc4915aebefce9835
electrs  | DEBUG - applying 1 new headers from height 102

❯ nigiri logs chopsticks
chopsticks  | time="2022-03-25T19:23:52Z" level=info msg="Starting server with configuration:" chain=bitcoin electrs_url="http://electrs:30000" faucet_enabled=true listen_url="0.0.0.0:3000" logger_enabled=true mining_enabled=true rpc_server_url="http://admin1:123@bitcoin:18443" tls_enabled=false
chopsticks  | time="2022-03-25T19:24:02Z" level=info msg="Faucet has been funded mining some blocks" num_blocks=101
chopsticks  | time="2022-03-25T19:27:10Z" level=info msg="New request from remote address: 172.27.0.1:60762" duration=124.517ms hostname="127.0.0.1:3000" method=POST path=/faucet response="{'txId':'7755e59b56b56c325d7a6632692ae8d521723fed8061da9f8c8917f3597871c7'}" start_time="2022-03-25T19:27:10Z" status=200
```

**Comment by tiero** [comment]
Nice catch! I think documentation/website is wrong. This should workd

```
nigiri logs bitcoin
```

**Comment by tiero** [comment]
Fixed #143
